packages/website/src/app/shared/loader.ts

```diff
@@ -125,20 +125,28 @@ export function getRepoPath(source: string): string {
 }
 
 export function replaceVariables(text: string, vars?: string) {
-  if (!vars) {
-    return text;
-  }
-  const variables = vars
-    .split(',')
-    .map((variable) => {
-      const [key, ...value] = variable.trim().split(/(:|%3[aA])/);
-      return Boolean(key) ? [key, decodeURIComponent(value.join(':'))] : undefined;
-    })
-    .filter(Boolean) as [string, string][];
-
-  for (const [key, value] of variables) {
-    // Replace $$key$$ with value
-    text = text.replaceAll(new RegExp(`\\$\\$${key}\\$\\$`, 'gm'), value);
+  if (vars) {
+    const variables = vars
+      .split(',')
+      .map((variable) => {
+        const decodedVariable = decodeURIComponent(variable);
+        const [key, ...value] = decodedVariable.trim().split(':');
+        return Boolean(key) ? [key, value.join(':')] : undefined;
+      })
+      .filter(Boolean) as [string, string][];
+
+    for (const [key, value] of variables) {
+      // Replace $$key$$ with value (but not \$$key$$)
+      console.log(`Replacing $$${key}$$ with "${value}"`);
```

Comment:
Question (for myself) @sinedied: what's the logging situation? Can you remove log in production or filter them?